frontend/src/component/playground/Playground/PlaygroundForm/PlaygroundConnectionFieldset/PlaygroundConnectionFieldset.test.tsx

```diff
@@ -150,7 +182,13 @@ test('should show an error when admin token', async () => {
 });
 
 test('should have a working clear button when token is filled', async () => {
-    render(<Component />);
+    const { container } = render(<Component />);
+    const projectAutocomplete = await screen.findByTestId(
+        'PLAYGROUND_PROJECT_SELECT',
+    );
+    const button = await within(projectAutocomplete).findByRole('button');
+    fireEvent.click(button);
+    await within(container).findByText('Default');
```

Comment:
Why are these lines suddenly added all over the place? Was the old way not sufficient?

Reply:
Because we need the project options to resolve from the rest call. `await within(container).findByText('Default');` ensures that the options are there before we try to do stuff